contracts/RitestreamNFT.sol

```diff
@@ -0,0 +1,99 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
```

Comment:
```suggestion
pragma solidity 0.8.4;
```
using the fixed solidity version was required by audit team, lets use 0.8.4 and keep all the same